pages/abi.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="abi-page">
+    <page-title :title="$t('abi-page.main-title')" />
+    <div class="block">
+      <tabs v-model="currentTab" :tabs="TABS_LIST" />
+      <div class="content">
+        <template v-for="tab in TABS_LIST" :key="tab.id">
+          <abi-encode-form
+            v-show="currentTab === TAB_IDS.encoder"
+            :title="tab.title"
+          />
+        </template>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { Tabs, PageTitle } from '#components'
+import { definePageMeta } from '#imports'
+import { AbiEncodeForm } from '@/forms'
+import { ref } from 'vue'
+import { i18n } from '~/plugins/localization'
+
+definePageMeta({
+  layout: 'solidity-tools',
+})
+
+enum TAB_IDS {
+  encoder = 'encoder',
+  decoder = 'decoder',
+}
+
+const { t } = i18n.global
+const TABS_LIST = [
```

Comment:
If the fields change in the tab type, there will be no error here. This is bad. It's better to make a Tab type and import it to determine the type of this array

Reply:
to discuss